src/api/app/controllers/trigger_controller.rb

```diff
@@ -42,6 +43,11 @@ def create
 
   private
 
+  def check_token_class
+    # It make no sense to process Token::Workflows on a runservice request
+    raise ActiveRecord::RecordNotFound, "Token not found" if @token.class == Token::Workflow
```

Comment:
I think this error message is somewhat confusing. Pointing users to the right endpoint would be helpful in this case.

Reply:
This check seems very similar to the one performed in this line, belonging to a yet not merged pull request: https://github.com/openSUSE/open-build-service/pull/15119/files#diff-77ed5d907b983934be02351099c20880d01bca6aab8366863839fc0721b3bb5fR61

Reply:
Well, we don't know if the user chose the wrong endpoint or the wrong token id, so pointing to another endpoint will make things more confusing if the token type is not the right one...

Reply:
> we don't know if the user chose the wrong endpoint or the wrong token id

True, I didn't think about this at first.